cellmerged.go

```diff
@@ -33,6 +33,24 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
 	return mergeCells, err
 }
 
+// RemoveMergeCell removes a MergeCell.
+func (f *File) RemoveMergeCell(sheet string, m MergeCell) error {
+	xlsx, err := f.workSheetReader(sheet)
+	if err != nil {
+		return err
+	}
+	if xlsx.MergeCells != nil {
+		for i := range xlsx.MergeCells.Cells {
+			ref := xlsx.MergeCells.Cells[i].Ref
+			if ref == m[0] {
+				f.deleteMergeCell(xlsx, i)
+				break
+			}
+		}
+	}
+	return nil
+}
+
```

Comment:
Please add the tests to cover your changes.

Reply:
ok.